pkg/services/authn/authnimpl/sync/user_sync.go

```diff
@@ -121,9 +121,14 @@ func (s *UserSync) FetchSyncedUserHook(ctx context.Context, identity *authn.Iden
 		return nil
 	}
 
+	orgID := r.OrgID
+	if orgID == 0 {
+		orgID = identity.OrgID
+	}
+
```

Comment:
This can probably break some thing, I think `orgID = 0` is used in some global contexts

Reply:
I made some tests without this part and it seems to be working as well. I think I can remove it and leave only the changes from the `s.userService.Create` func parameters from this file.